go-controller/pkg/util/libovsdb.go

```diff
@@ -94,7 +94,7 @@ func NewNBClientWithConfig(cfg config.OvnAuthConfig, stopCh <-chan struct{}) (cl
 		return nil, err
 	}
 
-	_, err = c.MonitorAll()
+	_, err = c.MonitorAll(context.TODO())
```

Comment:
@dcbw Should we use context.Background() instead?

Reply:
@flavio-fernandes I think here we'd want to create a WithCancel() context inside NewNBClientWithConfig() and then have a goroutine in there that:

```
ctx, cancel := context.WithCancel(context.TODO())
go func() {
    <-stopCh
    cancel()
}()

...

_,err = c.MonitorAll(ctx)
```

Or something like that. Then whenever we get around to passing a context down from the caller, we'd replace the TODO with that one.

Reply:
Ack! Done.

Reply:
@dcbw Does this apply to context passed to Transact as well? This might not be as handy as we don't have the stop channel readily available on the places we transact.